packages/patternfly-4/react-table/src/components/Table/Table.tsx

```diff
@@ -89,7 +89,7 @@ export interface IAction extends Omit<DropdownItemProps, 'title' | 'onClick'> {
   isSeparator?: boolean;
   itemKey?: string;
   title?: string | React.ReactNode;
-  onClick: (event: React.MouseEvent, rowIndex: number, rowData: IRowData, extraData: IExtraData) => void;
+  onClick: (event: React.MouseEvent, rowIndex: number | undefined, rowData: IRowData | undefined, extraData: IExtraData | undefined) => void;
```

Comment:
curious...does marking these params optional also resolve the issue? That would match the interface more closely.
```
onClick: (event: React.MouseEvent, rowIndex: number | undefined, rowData?: IRowData, extraData?: IExtraData) => void;
```

Reply:
either way is probably ok - but i think the long term goal is to share interfaces where possible